wiki/en/Running-a-Server.md

```diff
@@ -286,6 +286,12 @@ You can see the result of these commands if you run `systemctl status jamulus`,
 
 ---
 
+## Adding metadata to the Server
+
+You can add metadata to the welcome message of a Server to add additional, hidden information like contact information or policy requests for bots (this is comparable to [robots.txt](https://en.wikipedia.org/wiki/Robots.txt)). See the [Community Knowledge Base entry about metadata](/kb/2023/07/30/Server-Metadata.html) for more information.
+
+---
+
 ## Servers on the desktop
 
 Jamulus can run in Server mode in the graphical environment of a computer. This gives you a graphical user interface to control most of the settings. To do this, first [install Jamulus](Getting-Started) for your platform, then do one of the following:
```

Comment:
```suggestion
Jamulus can run in Server mode in the graphical environment of a computer. This gives you a graphical user interface to control most of the settings. To do this, first [install Jamulus for your platform](Getting-Started), then do one of the following:
```

Inline links need to be more than two words for much more successful discovery and useability. Links likely to wrap aren't so good. There's a sweet spot typically tied to the context.

Reply:
This should be in another PR. Please open an issue/pr

Reply:
What do you mean? This is a typical, small content improvement. Can you his "accept suggestion" and does that cause some workflow problem?

Reply:
The problem is that this PR might not even be merged and then your suggestion is lost

Reply:
Good point. It's a chance I'll take. I'll fix that whenever I see it, so if I don't get it this time, I'll get it next time.

Reply:
See #946